utils/_context/containers.py

```diff
@@ -546,10 +570,9 @@ def __init__(self, host_log_folder, *, use_proxy=True, environment=None) -> None
             host_log_folder=host_log_folder,
             environment=environment,
             healthcheck={
-                "test": f"curl --fail --silent --show-error --max-time 2 http://localhost:{self.agent_port}/info",
+                "test": f"curl --fail --silent --show-error --max-time 2 http://localhost:{self.apm_receiver_port}/info",
```

Comment:
I guess for the healthcheck the "localhost" works, right? It's only for external access

Reply:
yes, because it is executed **inside** the container